maintenance/projects/js-toolkit/packages/liferay-npm-build-tools-common/src/project/index.ts

```diff
@@ -362,18 +362,55 @@ export class Project {
 		}
 	}
 
-	_loadNpmbundlerrc(): void {
-		const npmbundlerrcPath = this._configFile.asNative;
+	_findAutopresets(): string[] {
+		const {dependencies, devDependencies} = this._pkgJson;
+
+		return Object.keys({
+			...(dependencies || {}),
+			...(devDependencies || {}),
+		}).reduce((autoPresets, pkgName) => {
```

Comment:
You could use default values when destructuring.
```suggestion
const { dependencies = {}, devDependencies = {} } = this._pkgJson;

return Object.keys({
  ...dependencies,
  ...devDependencies,
}).reduce((autoPresets, pkgName) => {

```